install/kubernetes/cilium/charts/agent/templates/NOTES.txt

```diff
@@ -1,5 +0,0 @@
-You have successfully installed Cilium {{ .Chart.Name }}
-
-Your release version is {{ .Chart.Version }}
-
-For any further help, visit https://docs.cilium.io/en/v{{ substr 0 3 .Chart.Version }}/gettinghelp
```

Comment:
Is this intended to be replaced by another `NOTES.txt` file or is this deliberately omitted?

Reply:
I simpley hadn't migrated the original notes over yet.